content/ko/docs/concepts/configuration/assign-pod-node.md

```diff
@@ -0,0 +1,399 @@
+          requiredDuringSchedulingIgnoredDuringExecution:
+          - labelSelector:
+              matchExpressions:
+              - key: app
+                operator: In
+                values:
+                - web-store
+            topologyKey: "kubernetes.io/hostname"
+        podAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+          - labelSelector:
+              matchExpressions:
+              - key: app
+                operator: In
+                values:
+                - store
+            topologyKey: "kubernetes.io/hostname"
+      containers:
+      - name: web-app
+        image: nginx:1.12-alpine
+```
+
+만약 위의 두 디플로이먼트를 생성하면 세 개의 노드가 있는 클러스터는 다음과 같아야 한다.
+
+|       node-1         |       node-2        |       node-3       |
+|:--------------------:|:-------------------:|:------------------:|
+| *webserver-1*        |   *webserver-2*     |    *webserver-3*   |
+|  *cache-1*           |     *cache-2*       |     *cache-3*      |
+
+여기서 볼 수 있듯이 `web-server` 의 세 레플리카들이 기대했던 것처럼 자동으로 캐시와 함께 위치하게 된다.
+
+```
+kubectl get pods -o wide
+```
+출력은 다음과 유사할 것이다:
+```
+NAME                           READY     STATUS    RESTARTS   AGE       IP           NODE
+redis-cache-1450370735-6dzlj   1/1       Running   0          8m        10.192.4.2   kube-node-3
+redis-cache-1450370735-j2j96   1/1       Running   0          8m        10.192.2.2   kube-node-1
+redis-cache-1450370735-z73mh   1/1       Running   0          8m        10.192.3.1   kube-node-2
+web-server-1287567482-5d4dz    1/1       Running   0          7m        10.192.2.3   kube-node-1
+web-server-1287567482-6f7v5    1/1       Running   0          7m        10.192.4.3   kube-node-3
+web-server-1287567482-s330j    1/1       Running   0          7m        10.192.3.2   kube-node-2
+```
+
+##### 절대 동일한 노드에 위치시키지 않게 하기
+
+위의 예시에서 `topologyKey:"kubernetes.io/hostname"` 과 함께 `PodAntiAffinity` 규칙을 사용해서
+두 개의 인스터스가 동일한 호스트에 있지 않도록 redis 클러스터를 배포한다.
+같은 기술을 사용해서 고 가용성을 위해 안티-어피니티로 구성된 스테이트풀셋의 예시는
+[ZooKeeper 튜토리얼](/docs/tutorials/stateful-application/zookeeper/#tolerating-node-failure)을 본다.
+
+## nodeName
+
+`nodeName` 은 가장 간단한 형태의 노트 선택 제약 조건이지만,
+한계로 인해 일반적으로는 사용하지 않는다.
+`nodeName` 은 PodSpec의 필드이다. 만약 비어있지 않으면, 스케줄러는
+파드를 무시하고 명명된 노드에서 실행 중인 kubelet이
+파드를 실행하려고 한다. 따라서 만약 PodSpec에 `nodeName` 가
+제공된 경우, 노드 선텍을 위해 위의 방법보다 우선한다.
+
+`nodeName` 을 사용해서 노드를 선택할 때의 몇 가지 제한은 다음과 같다.
+
+-   만약 명명된 노드가 없으면, 파드가 실행되지 않고
+    따라서 자동으로 삭제될 수 있다.
+-   만약 명명된 노드에 파드를 수용할 수 있는
+    리소스가 없는 경우 파드가 실패하고, 그 이유는 다음과 같이 표시된다.
+    예: OutOfmemory 또는 OutOfcpu.
+-   클라우드 환경의 노드 이름은 항상 예측 가능하거나
+    안정적인 것은 아니다.
+
+여기에 `nodeName` 필드를 사용하는 파드 설정 파일 예시가 있다.
+
+```yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: nginx
+spec:
+  containers:
+  - name: nginx
+    image: nginx
+  nodeName: kube-01
+```
+
+위 파드는 kube-01 노드에서 실행될 것이다.
+
+{{% /capture %}}
+
+{{% capture whatsnext %}}
+
+[테인트(Taints)](/docs/concepts/configuration/taint-and-toleration/)는 노드가 특정 파드들을 *쫓아내게* 할 수 있다.

```

Comment:
```suggestion
[테인트](/docs/concepts/configuration/taint-and-toleration/)는 노드가 특정 파드들을 *쫓아내게* 할 수 있다.
```
Since we inform the original English once for a word in a document, we don't need to provide English word here again.
In [153 line](https://github.com/kubernetes/website/pull/18955/files#diff-64362a813eb71f90aeb59840e0d4dc2cR153), Taint was firstly translated to "테인트(taint)" already.
I think this comment will resolve @sftim 's correction as well.